Makefile

```diff
@@ -49,6 +49,8 @@ install:
 	sed -e 's_^\(ExecStart=/usr/share/openqa/script/worker\) \(--instance %i\)$$_\1 --no-cleanup \2_' \
 		systemd/openqa-worker@.service \
 		> "$(DESTDIR)"/usr/lib/systemd/system/openqa-worker-no-cleanup@.service
+	sed -i '/Wants/aConflicts=openqa-worker@.service' \
```

Comment:
I don't think you can conflict with templates. After all worker@1 and worker-no-cleanup@2 do not conflict